content/13.conclusion.md

```diff
@@ -1 +1,7 @@
 ## Conclusion {#conclusion}
+
+Deep learning techniques have the potential for wide use in biology, meeting or exceeding the performance of both humans and the current state-of-the art algorithms in a variety of tasks.
+Beyond simply achieving performance, deep learning has the potential to generate novel biological insights that may then be applied back to fundamental research.
```

Comment:
I don't think "simply achieving performance" is correct -- I don't think someone/something can "achieve performance." You meant "good predictive performance?"